feat: add `flagd` CRD with ingress support (#633)
open-feature/open-feature-operator: chart/open-feature-operator/README.md

| `controllerManager.manager.resources.requests.cpu`                        | Sets cpu resource requests for operator.                                                                                         | `10m`                                        |
| `controllerManager.manager.resources.requests.memory`                     | Sets memory resource requests for operator.                                                                                      | `64Mi`                                       |
| `controllerManager.replicas`                                              | Sets number of replicas of the OpenFeature operator pod.                                                                         | `1`                                          |
| `managerConfig.flagsValidatonEnabled`                                     | Enables the validating webhook for FeatureFlag CR.                                                                               | `true`                                       |

> | `managerConfig.flagdResourceEnabled`                                      | Enables the controller for the Flagd CR and adds the required permissions to create/update/delete Service and Ingress resources. | `true`                                       |

Reviewer: ```suggestion
| `managerConfig.flagdResourceEnabled`                                      | Enables the controller for the Flagd CR and adds the required permissions to automatically manage the exposure of flagd via Service and Ingress resources. | `true`                                       |
```

Maybe you or somebody else can word this better, but I think a "why" was missing here, and I've tried to capture that.

Author: thanks for the suggestion, this makes things a bit clearer
